Gemfile

```diff
@@ -37,6 +37,9 @@
 gem "net-smtp", "~> 0.3.1"
 gem "psych", "~> 4.0"
 
+gem "pronto", "~> 0.11"
+gem 'pronto-rubocop', require: false
+
```

Comment:
Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `faraday-retry` should appear before `pronto-rubocop`.